awx/api/serializers.py

```diff
@@ -413,6 +419,16 @@ def get_modified(self, obj):
             return obj.modified
         return None
 
+    def get_extra_kwargs(self):
```

Comment:
@rooftopcellist The implementation of this function might give you some hints on getting `token` and `refresh_token` hidden after creation. It's basically a hack on DRF source.